package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@privy-io/react-auth": "1.92.6",
+    "@privy-io/react-auth": "^2.0.0",
```

Comment:
Does this need to be pinned?

Reply:
Oh I think renovate will handle that actually.

Reply:
yeah eventually it will